test/e2e/helpers.js

```diff
@@ -100,22 +100,55 @@ async function withFixtures(options, testSuite) {
   let webDriver;
   let driver;
   let failed = false;
+
+  let ganacheServer;
+  let anvilServer;
+
   try {
-    if (!disableGanache) {
```

Comment:
instead of using this argument into `withFixtures` we now have the localNode: 'none' option, when we want to signal that we want no local node server running -- see below